platform/engineering/collab-cycle/architecture-intent-template.md

```diff
@@ -0,0 +1,56 @@
+# Architecture Intent Template
+
+__*** DRAFT *** DRAFT *** DRAFT ***__
+
+The Architecture Intent meeting helps your team build a solution that meets Veteran-facing Service Platform (VSP) engineering standards and lowers the potential for launch-blocking issues later in the development cycle.
+
+## What is the purpose of Architecture Intent meeting?
+
+The Architecture Intent meeting is less a formal presentation and more of a discussion.  It not only provides OCTO-DE and Platform with an early understanding of the product/feature your team wants to build, but is also an opportunity to collaborate and provide feedback on the intended implementation and surface any adjustments needed to meet VSP engineering standards. The focus is on making sure your code meets user needs within the constraints of the platform your building on.
+
+## Preparing for the Architecture Intent meeting
+
+You should write up your Architecture Intent using the template below.The Governance Team will need at least **2 business days** to review your matierials.
+
+## Architecture Intent Template
+
+Your document should be brief and high-level.  One to two pages is appropriate for many changes.  Focus on the high level and link to supporting material where appropriate; this is not a detailed engineering spec.
+
+- Product description
```

Comment:
I love this section, where should we have them live?

Reply:
Added a TODO to figure out a place for these docs.  Will address in the next draft.